src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,89 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        System.out.println("FILLING MANUFACTURERS DB:");
+        List<Manufacturer> manufacturers = List.of(
+                new Manufacturer(null, "VW", "Germany"),
+                new Manufacturer(null, "Ford", "US"),
+                new Manufacturer(null, "Hyundai", "South Korea"),
+                new Manufacturer(null, "Toyota", "Japan"));
```

Comment:
I think we should create constructor without id for this purpouse

Reply:
Done